ddtrace/_monkey.py

```diff
@@ -84,24 +82,22 @@
     "dogpile_cache": True,
     "yaaredis": True,
     "asyncpg": True,
+    "tornado": False,
 }
 
+
+CONTRIB_DEPENDENCIES = {
+    "tornado": ("futures",),
+}
+
+
 _LOCK = threading.Lock()
 _PATCHED_MODULES = set()
 
-# Modules which are patched on first use
-# DEV: These modules are patched when the user first imports them, rather than
-#      explicitly importing and patching them on application startup `ddtrace.patch_all(module=True)`
-# DEV: This ensures we do not patch a module until it is needed
-# DEV: <contrib name> => <list of module names that trigger a patch>
-_PATCH_ON_IMPORT = {
-    "aiohttp": ("aiohttp",),
-    "aiobotocore": ("aiobotocore",),
-    "celery": ("celery",),
-    "flask": ("flask",),
-    "gevent": ("gevent",),
-    "requests": ("requests",),
-    "botocore": ("botocore",),
+# Module names that need to be patched for a given integration. If the module
```

Comment:
How was `snowflake.connector` getting patched prior to this change given that this dict didn't exist?